setup/main.go

```diff
@@ -74,6 +75,10 @@ func main() {
 
 	app.Command("apply", "Create/Update a Layer0", func(cmd *cli.Cmd) {
```

Comment:
`dockercfg` is not a flag for the `context` object, but for the `apply` command. This should look like:
```
app.Command("apply", "Create/Update a Layer0", func(cmd *cli.Cmd) {
		flags := loadFlags(cmd, []string{"access_key", "secret_key", "region"})
		force := cmd.BoolOpt("force", false, "Set this flag to skip prompting on a missing dockercfg file")
		dockercfg := cmd.StringOpt("dockercfg", "", "Path to valid dockercfg file")

		vpc := cmd.StringOpt("vpc", "", "VPC id to target.  Will create new VPC if blank.")
		flags["vpc_id"] = vpc

		command := func(c *context.Context) error {
			return context.Apply(c, *force, *dockercfg)
		}

		flagsCommand(cmd, command, flags)
	})
```